lang/test/org/partiql/lang/eval/evaluatortestframework/ExpectedResultFormat.kt

```diff
@@ -8,34 +8,17 @@ package org.partiql.lang.eval.evaluatortestframework
  */
 enum class ExpectedResultFormat {
     /**
-     * The expected value is expressed in Ion and Ion's equivalence is used to assert the correct option.  This is the
-     * strictest (and preferred) option when the result isn't a bag because it doesn't support PartiQL's BAG semantics,
-     * thus, a query might have the correct result but in a different order the expected value, this would cause a
-     * false negative.
+     * Transform the compiler's actual output [ExprValue] to [IonValue], and then compare it with the specified
+     * expected [IonValue]. It can cause false negative when comparing PartiQL bags, since order matters in Ion list
      */
     ION,
```

Comment:
This is still used in some tests which are explicitly testing the `IonValue` representation of an `ExprValue`.